### DISCUSSION TITLE
sent iota to the fromer receiving adress

### DISCUSSION THREAD

**Discussion by akinomoni** [iotaledger/firefly#2232] [Help & Support] [answered]
Hi. I made  staking for some IOta one month ago. To do so, I got a receiving adress. Everything ok. I asked my sister to send me some Iota she bought for me, but she could not send them, as Binance didn't allow the withdraw that day. Then when she was allow to withdraw it was some day later. She didn't know and she sent me to the receiving adress I got that day. When I got in to put those new IOta staking I realised that I had a new receiving adress. can I recover those IOta sent to the former adress, please? can somebody help me?

thank you

Monika

**Comment by cometax** [comment, marked as answer]
The iota she Sent should appear in the wallet even if the receiving address shown is different. Once you verify you have the iotas in the wallet, go to the stake tab and try to merge the stake so that you stake the full amount

**Reply by akinomoni** [reply]
Thank you so much cometax, I did it! I had to connect my ledger and here they were. Bravo! I love you!  :)

**Comment by akinomoni** [comment]
thank you very much for your help. I'm a learner, I connect my ledger and
they appeared, sorry for disturbing you. You made me really happy!  :))

good luck!


Monika
